js/average.js

```diff
@@ -0,0 +1,46 @@
+
+function total(array){
+    var total = array.reduce(function (accumulator, currentValue) {
```

Comment:
Beautiful! Just FYI, you can just return this array.reduce on line 3 rather than 6. They way you did this was great too😊